tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,518 @@
+      "req_list": [
+        {
+          "name": "Classics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Classics uses knowledge of Greek or Latin (or both languages) as a gateway to the study of the literature, history, and culture of ancient Greece and Rome.\n\nStudents are expected to pass a sight translation examination from Greek or Latin. This examination may be taken, by arrangement, at the end of any term in the junior or senior year; it will be graded pass/fail.\n\nStudents who are considering further work in the field, either in graduate school or in independent study, should take both Latin and Greek to the 300 level, continuing with both languages in each term of the junior and senior years. Such students are also strongly advised to take at least one course in Greek history and one in Roman history in their underclass years.",
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "To enter this program, a student normally should have completed CLG 108 or LAT 108 or demonstrated intermediate proficiency in Greek or Latin through test scores (SAT, AP), a placement exam, or coursework. A strongly motivated student who has completed CLG 102 or LAT 102 (or CLG 103 or LAT 103) may concentrate, with permission of the departmental representative.",
+              "double_counting_allowed": true,
+              "course_list": [
+                "CLG 108",
+                "LAT 108",
+                "CLG 102",
+                "LAT 102",
+                "CLG 103",
+                "LAT 103"
+              ]
+            },
+            {
+              "name": "Departmentals",
+              "max_counted": 1,
+              "min_needed": 8,
+              "explanation": "Eight departmental courses are required. Of these, five must be in the original languages at the 200 level or above, including at least one course at the 300 level. The combination of LAT 104-108, or CLG/LAT 105-108, may be counted as the equivalent of one 200-level course. One course in ancient history (CLA 214, 216, 217, 218, or 219, or HIS 343) must also be included among the departmentals. Students must also take the Junior Seminar in the fall of their junior year.",
+              "req_list": [
+                {
+                  "name": "Original Languages",
+                  "max_counted": null,

```

After playing with it a bit, it seems that this should be limited to 7 so that it doesn't all 8 courses (students still have to satisfy the ancient history requirement).
```suggestion
                  "max_counted": 7,
```